src/components/Tool/NavToolList/NavToolList.test.tsx

```diff
@@ -12,16 +14,48 @@ const router = {
   isReady: true,
 };
 
+const toggleMock = jest.fn();
+
+const TestComponent = () => (
+  <ThemeProvider theme={theme}>
+    <TestRouter router={router}>
+      <GqlMockedProvider>
+        <NavToolList toggle={toggleMock} isOpen={true} />
+      </GqlMockedProvider>
+    </TestRouter>
+  </ThemeProvider>
+);
+
 describe('NavToolList', () => {
```

Comment:
I think it would be great to add a test that mocks the number of things that need fixed and make sure that those counts render.